provider/inspec/common~copy.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# These files are okay to be copied "AS-IS" to the module's final destination:
+
+'libraries/google/hash_utils.rb': 'google/hash_utils.rb'
```

Comment:
Why do we need these files? I didn't see any of the utilities used anywhere?

Reply:
Ended up using hash utils in plural resource.

Reply:
What about string_utils?